azure-pipelines.yml

```diff
@@ -74,12 +74,9 @@ jobs:
       queue: Windows.10.Amd64.ClientRS5.Open
       csproj: src\benchmarks\micro\MicroBenchmarks.csproj
       runCategories: 'coreclr corefx' 
-      frameworks: # for public jobs we want to make sure that the PRs don't break any of the supported frameworks
-        - netcoreapp5.0
-        - netcoreapp3.1
-        - netcoreapp3.0
-        - netcoreapp2.2
-        - netcoreapp2.1
+      channels:
```

Comment:
with this change, we test all frameworks on Ubuntu and only two on Windows. Is this intentional?

https://github.com/dotnet/performance/blob/8b9c22bb7d75e966260c89eccf02ae015ea87b21/azure-pipelines.yml#L151-L156

Reply:
Good catch! This is a mistake. I'll update it asap.